pkgs/memory_usage/example/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: memory_usage_example
+publish_to: none
+
+environment:
+  sdk: ^3.0.0
+
+dependencies:
+  memory_usage: ^0.1.0
+
+dependency_overrides:
+  flutter_lints: ^3.0.0
```

Comment:
You'll want to move the linting dependency into a `dev_dependency` section.

Reply:
You'll need to create a new section here - `dev_dependency`, have flutter_lints specified there, and retain the `dependency_overrides` section for use by the `memory_usage` dep.

Reply:
I do not know how it continue squeezing to the PR. Fixed.